tests/utils/versiontools_test.py

```diff
@@ -131,7 +134,9 @@ def test_get_version_from_scm_tag_in_git_repo(
         git_cmd, git_commit_cmd, git_tag_cmd,
 ):
     """Check that get_version_from_scm_tag works properly in Git repo."""
-    assert get_self_version() == '0.1.dev0'
+    # setuptools_scm 8+ includes a date stamp here.
```

Comment:
I'd like to understand if there's a parameter we can set that would normalize its behavior and keep it as it was...